spec/models/timer_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe Timer, type: :model do
+
+  let(:player) { FactoryGirl.create(:player, current_player.id, email: 'pam@rook.com', password: 'BESTKITTY') }
+  let(:game) { FactoryGirl.create(:game, game.id)}
+
+    it "should determine when a player starts" do 
+      expect(DateTime.new).to be_a(DateTime)
+    end
+
+    it "should determine how much time the players have" do
+      # expect(@time_left).to eq(Integer)
```

Comment:
Kelly - this worked for these failing tests (yea!) But I still have 2 failing tests because I added an after_initialize callback - ugh.